plugins/ui/src/js/src/styles.scss

```
@@ -90,3 +90,11 @@
   overflow: hidden;
   width: 100%;
 }
+
+.ui-widget-error-contextual-help {
+  section[class*='spectrum-ContextualHelp-dialog'] {
+    // Our error messages can be quite large. The default size of the contextual help is only 250px and is too small.
+    // Just set a size automatically based on the content.
+    width: auto;
```

```suggestion
    width: fit-content;
```

Basically the same, just what spectrum declares further down in precedence for regular dialogs, instead of auto.